resources/config/services/services.yml

```diff
@@ -21,6 +21,17 @@ services:
     bind:
       $showSubmissionCount: '%application.show_submission_count%'
 
+  OpenCFP\Http\Action\Forgot\IndexAction:
+    public: true
+    bind:
+      $resetForm: '@OpenCFP\Http\Form\ForgotForm'
+
+  OpenCFP\Http\Form\ForgotForm:
```

Comment:
@derrabus 

Since I believe you are the one who has the most experience with Symfony, can you take a look and suggest a better way to wire up an actual form from a form type, if you can think of one?